README.md

```diff
@@ -4,31 +4,44 @@ This repository both bootstraps and manages the deployment lifecycle of Lodestar
 
 ## Bootstrapping
 
-To create an instance of Lodestar in a cluster, apply the `bootstrap` Helm chart using the following command:
+To create an instance of Lodestar in a cluster, ensure you're on the branch/commit-id of your choice and you have working oc/ku>
 
 ```sh
-helm install lodestar bootstrap/ --set application.ref=<desired git ref>
+kustomize build bootstrap/ | oc apply -f -
 ```
 
-Make sure to replace `<desired-git-ref>` with the tag that you want to deploy to this cluster. If this is a staging cluster, you might have a Git tag such as `deploy-staging`; and if this is a production cluster, maybe `deploy-production`.
+NOTE: Running `kustomize build bootstrap/` alone will print manifests that can be reviewed before actually applying it to the c>
 
 ## Lifecycle Management
 
-Lodestar is not a single application, but a collection of components (or services) that all add up to a working system. Those services might be versioned separately from one another. Therefore, a "deployment of Lodestar" is really a manifest of the versions of its constituent components that are known to work well with each other.
+Lodestar is not a single application, but a collection of components (or services) that all add up to a working system. Those s>
 
-This manifest lives in `applications/values.yaml`:
+Kustomize patches for respective components live in `bootstrap/patches/` directory:
+
+Example file, where `v1.3.1` is target version, `bootstrap/patches/lodestar-backend.yaml`:
```

Comment:
Not a fan of keeping an example file in here as it will easily get out of sync with the actual file (not just the version). Perhaps just explain that the versions are kept in the patch files, and no need to show an example here.